pkg/webui/account/views/landing/index.js

```diff
@@ -18,7 +18,7 @@ import classnames from 'classnames'
 
 import authRoutes from '@account/constants/auth-routes'
 
-import sidebarStyle from '@ttn-lw/components/navigation/side/side.styl'
+import sidebarStyle from '@ttn-lw/components/sidebar/side-menu/side.styl'
```

Comment:
I've moved the `navigation/side` into the sidebar component. I believe this is the only place we should ever use this navigation list in. There are currently some areas where we use it outside of the sidebar (I believe in the Account app), but we will address this later.

Reply:
Please, also update the path in the side-menu story (`pkg/webui/console/containers/side-bar/story.js`)